src/lib/features/feature-lifecycle/feature-lifecycle.e2e.test.ts

```diff
@@ -115,6 +112,10 @@ const expectFeatureStage = async (featureName: string, stage: StageName) => {
     });
 };
 
+const getFeaturesLifecycleCount = async () => {
+    return app.request.get(`/api/admin/lifecycle/count`).expect(200);
```

Comment:
I'm wondering if this is the correct name for the new API

Reply:
I can't think of anything better